backend/amd64/proc.ml

```diff
@@ -646,6 +650,38 @@ let frame_required ~fun_contains_calls ~fun_num_stack_slots =
 let prologue_required ~fun_contains_calls ~fun_num_stack_slots =
   frame_required ~fun_contains_calls ~fun_num_stack_slots
 
+(* returned size includes return address *)
+let frame_size ~stack_offset ~fun_contains_calls ~fun_num_stack_slots =
+  if frame_required ~fun_contains_calls ~fun_num_stack_slots then begin
+    let sz =
```

Comment:
(As briefly discussed off-line, that
code should be updated to take
into account the support for SIMD.)

Reply:
This should now be fixed